### PR TITLE
feat: digital credentials detailed instructions

### PR DESCRIPTION
**Pull request by amanji**
*Issue #:* bcgov/entity#18394

*Description of changes:*

* Adds details instructions for getting a digital credential
* Better edge case handling when revoking/removing a credential

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/DigitalCredentials/CredentialsDetailSteps.vue [hunk 1]
```diff
@@ -0,0 +1,139 @@
+<template>
+  <div>
```
Comment (severinbeauvais) [inline src/components/DigitalCredentials/CredentialsDetailSteps.vue:2]: Id, please.

**Review by severinbeauvais — APPROVED** [review]

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-filings-dev--pr-575-nvq4ivk6.web.app

**Comment by severinbeauvais** [comment]
I had a quick look at the changes at the preview site above. Looks amazing! Good to merge when PR is ready.

**Comment by amanji** [comment]
The test failures is from the GitHub action unable to upload the coverage report. The tests are passing.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-filings-dev--pr-575-nvq4ivk6.web.app

**Comment by severinbeauvais** [comment]
> The test failures is from the GitHub action unable to upload the coverage report. The tests are passing.

When that happens, open the failing check's details and click "Re-run jobs" (failed jobs only) at the top. I've done that already here.

**Pull request merged**